pkg/util/inplaceupdate/inplace_update_defaults.go

```diff
@@ -265,16 +266,24 @@ func defaultCalculateInPlaceUpdateSpec(oldRevision, newRevision *apps.Controller
 			}
 			return nil
 		}
-		if op.Operation != "replace" || !containerImagePatchRexp.MatchString(op.Path) {
-			return nil
+		if op.Operation == "replace" && containerImagePatchRexp.MatchString(op.Path) {
```

Comment:
Why the modification? Does it have anything to do with validate?

Reply:
ignore ...